master/reference/calicoctl/patch.md

```diff
@@ -0,0 +1,121 @@
+---
+title: calicoctl patch
+canonical_url: 'https://docs.projectcalico.org/master/reference/calicoctl/commands/patch'
```

Comment:
I think this is what is causing your CI to fail - it should pass once merged (this URL doesn't exist yet).

Reply:
That's what I thought, but just wanted to check!